### PR TITLE
Toolsdocs

### PR DESCRIPTION
**Pull request by katejeffreys**

### CODE REVIEW

File: app/2.0/docs/tools/polymer-json.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  "entrypoint": "index.html",
+  "shell": "src/shop-app.html",
+  "fragments": [
+    "src/shop-list.html",
+    "src/shop-detail.html",
+    "src/shop-cart.html",
+    "src/shop-checkout.html",
+    "src/lazy-resources.html"
+  ],
+  "sourceGlobs": [
```
Comment (abdonrd) [inline app/2.0/docs/tools/polymer-json.md]: `sourceGlobs` > `sources`

File: app/2.0/docs/tools/polymer-json.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+Dependencies that the analyzer component of the Polymer build toolchain can’t discover, and that do not need to be bundled.
+
+### shell
+*Required* 
+Type: `String`
+
+The app shell. Contains the main code that boots the app and loads necessary resources. The shell usually includes the common UI for the app, and the router, which loads resources needed for the current route.
+
+### sourceGlobs
```
Comment (abdonrd) [inline app/2.0/docs/tools/polymer-json.md]: `sourceGlobs` > `sources`

File: app/2.0/docs/tools/polymer-json.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  |-index.html
+  |-manifest.json
+  |-polymer.json
+  |-service-worker.js
+  |-src
+  |-sw-precache-config.js
+  |-test
+```
+
+The `sourceGlobs` property is set as follows:
```
Comment (abdonrd) [inline app/2.0/docs/tools/polymer-json.md]: `sourceGlobs` > `sources`

File: app/2.0/docs/tools/polymer-json.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+```json
+"sourceGlobs": [
```
Comment (abdonrd) [inline app/2.0/docs/tools/polymer-json.md]: `sourceGlobs` > `sources`

File: app/2.0/docs/tools/polymer-json.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+    "css": {"minify": true},
+    "html": {"minify": true}
+  }]
+```
+
+And here is a configuration to generate two optimized builds: One bundled and one unbundled:
+
+```json
+"builds": [{
+    “name”: “bundled”,
```
Comment (abdonrd) [inline app/2.0/docs/tools/polymer-json.md]: `“name”: “bundled”,` > `"name": "bundled",`

File: app/2.0/docs/tools/polymer-json.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+    "bundle": true,
+    "js": {"minify": true},
+    "css": {"minify": true},
+    "html": {"minify": true}
+  },{
+    “name”: “unbundled”,
```
Comment (abdonrd) [inline app/2.0/docs/tools/polymer-json.md]: `“name”: “unbundled”,` > `"name": "unbundled",`

**Comment by katejeffreys** [comment]
@abdonrd Thanks for catching these :) I have included fixes in the new PR.

**Pull request closed without merging**